src/command/Commands.js

```diff
@@ -117,7 +117,6 @@ define(function (require, exports, module) {
     exports.CMD_ADD_TO_WORKINGSET_AND_OPEN  = "cmd.addToWorkingSetAndOpen";          // DocumentCommandHandlers.js   handleOpenDocumentInNewPane()
 
     // NAVIGATE
-    exports.NAVIGATE_NEXT_DOC           = "navigate.nextDoc";           // DocumentCommandHandlers.js   handleGoNextDoc()
```

Comment:
We can't do it this way, since this is only going to fix part of it.  This file is defining a module with the default set of Commands.  Elsewhere in the code, you'll see that there are users of this:

```
$ git grep NAVIGATE_NEXT_DOC 
src/command/Commands.js:    exports.NAVIGATE_NEXT_DOC           = "navigate.nextDoc";           // DocumentCommandHandlers.js   handleGoNextDoc()
src/command/Commands.js:    exports.NAVIGATE_NEXT_DOC_LIST_ORDER    = "navigate.nextDocListOrder";           // DocumentCommandHandlers.js   handleGoNextDocListOrder()
src/command/DefaultMenus.js:        menu.addMenuItem(Commands.NAVIGATE_NEXT_DOC);
src/command/DefaultMenus.js:        menu.addMenuItem(Commands.NAVIGATE_NEXT_DOC_LIST_ORDER);
src/command/Menus.js:        NAVIGATE_DOCUMENTS_COMMANDS:        {sectionMarker: Commands.NAVIGATE_NEXT_DOC},
src/document/DocumentCommandHandlers.js:    CommandManager.register(Strings.CMD_NEXT_DOC,                    Commands.NAVIGATE_NEXT_DOC,              handleGoNextDoc);
src/document/DocumentCommandHandlers.js:    CommandManager.register(Strings.CMD_NEXT_DOC_LIST_ORDER,         Commands.NAVIGATE_NEXT_DOC_LIST_ORDER,   handleGoNextDocListOrder);
src/extensions/default/NavigationAndHistory/main.js:        KeyBindingManager.addBinding(Commands.NAVIGATE_NEXT_DOC, KeyboardPrefs[NEXT_IN_RECENT_FILES]);
test/perf/OpenFile-perf-files/brackets-concat.js:    exports.NAVIGATE_NEXT_DOC           = "navigate.nextDoc";
test/perf/OpenFile-perf-files/brackets-concat.js:        CommandManager.register(Strings.CMD_NEXT_DOC,           Commands.NAVIGATE_NEXT_DOC, handleGoNextDoc);
test/perf/OpenFile-perf-files/brackets-concat.js:        KeyBindingManager.addBinding(Commands.NAVIGATE_NEXT_DOC, [{key: "Ctrl-Tab",   platform: "win"},
test/perf/OpenFile-perf-files/brackets-concat.js:    exports.NAVIGATE_NEXT_DOC           = "navigate.nextDoc";
test/perf/OpenFile-perf-files/brackets-concat.js:        CommandManager.register(Strings.CMD_NEXT_DOC,           Commands.NAVIGATE_NEXT_DOC, handleGoNextDoc);
test/perf/OpenFile-perf-files/brackets-concat.js:        KeyBindingManager.addBinding(Commands.NAVIGATE_NEXT_DOC, [{key: "Ctrl-Tab",   platform: "win"},
```

So you'd have to deal with all the various call sites too.  A better solution would be to just not add the keybinding.

Also, if you do this for NEXT, you should probably do it for PREV too.